fms/fms.F90

```diff
@@ -202,8 +202,8 @@ module fms_mod
           open_file, open_direct_file
 
 ! routines for reading/writing distributed data
-public :: set_domain, read_data, write_data, read_compressed, read_distributed
-public :: get_domain_decomp, field_size, nullify_domain
```

Comment:
You forgot to bring these back. 

Most model components that use this have 
`use          fms_mod, only: set_domain, nullify_domain`

Reply:
You're right!  Thanks for catching this.